test/e2e/azure_logcollector.go

```diff
@@ -20,12 +20,16 @@ package e2e
 
 import (
 	"context"
+	"fmt"
 	"io/ioutil"
 	"net/http"
 	"path/filepath"
-	"sigs.k8s.io/cluster-api-provider-azure/azure"
 	"strings"
 
+	"sigs.k8s.io/cluster-api-provider-azure/api/v1alpha4"
+
```

Comment:
nit: remove extra new line

Reply:
interesting goimports didn't fix this?  I also thought we ran go imports in the validation scripts